lib/ansible/modules/remote_management/manageiq/manageiq_alert_profiles.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2017 Red Hat Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+
+module: manageiq_alert_profiles
+
+short_description: Configuration of alert profiles for ManageIQ
+extends_documentation_fragment: manageiq
+version_added: '2.5'
+author: Elad Alfassa (ealfassa@redhat.com)
+description:
+  - The manageiq_alert_profiles module supports listing, adding, updating and deleting alert profiles in ManageIQ.
+
+options:
+  state:
+    description:
+      - absent - alert profile should not exist,
+      - present - alert profile should exist,
+      - list - return a list of alert policies.
```

Comment:
typo?  "list of alert profiles"?

Reply:
yep, it was a typo. Fixed now. (I'll squash the commits before this PR will be merged)